src/net.rs

```diff
@@ -45,6 +48,7 @@ pub enum RequestAction {
     NewRoom(String),
     JoinRoom(String),
     LeaveRoom,
+    QuitGame,
```

Comment:
Isn't this already covered by `LeaveRoom`?

Reply:
`LeaveRoom` keeps the player connected and in lobby. `QuitGame` would tell the Server that this Client has decided to gracefully terminate the connection.